assets/js/sections/common.js

```diff
@@ -265,6 +265,28 @@ function qso_edit(id) {
                         selectize_usa_county('#stateDropdown', '#stationCntyInputEdit');
                     }
 
+                    var unsupported_lotw_prop_modes = [];
+                    $.ajax({
+                       url: base_url + 'index.php/qso/unsupported_lotw_prop_modes',
+                       type: 'get',
+                       async: false,
+                       success: function(data) {
+                          unsupported_lotw_prop_modes = $.parseJSON(data);
+                       },
+                    });
+
+                    $('#prop_mode').change(function(){
+                       if (unsupported_lotw_prop_modes.includes($('#prop_mode').val())) {
+                          $('#lotw_sent').prop('disabled', true);
+                          $('#lotw_rcvd').prop('disabled', true);
+                          $('#lotw_propmode_hint').html("This propagation mode is not supported by LoTW. LoTW sent/received fields disabled.").fadeIn("slow");
```

Comment:
@HB9HIL I need a little support with translations here. Can u pls give a hint on how to integrate into JS?

Reply:
Just as we did before. Just create a JS var in PHP. Thats it

Reply:
Just as we did before. Just create a JS var in PHP. Thats it

Reply:
Will do